go/enclave/rollupmanager/rollupmanager.go

```diff
@@ -129,9 +129,30 @@ func (re *rollupManager) CreateRollup() (*core.Rollup, error) {
 		return nil, fmt.Errorf("current head batch matches the rollup head bash")
 	}
 
-	newRollup := createNextRollup(rollup, batches)
+	limiter := RollupLimiter(MaxTransactionSize)
 
-	if err := re.l2chain.SignRollup(newRollup); err != nil {
+	extBatches := make([]*common.ExtBatch, 0)
+	for _, batch := range batches {
+		extBatch := batch.ToExtBatch(re.TransactionBlobCrypto)
+		err := limiter.Consume(extBatch)
+		if err != nil && errors.Is(err, ErrFailedToEncode) {
+			panic("cannot rlp encode batch. l1 updates will stall.")
```

Comment:
might want to avoid doing a panic here

Reply:
Not sure how can we recover from having a batch that doesn't rlp encode, the system will just keep running in a stalled state and potentially do something even worse cause it can't produce a rollup

Reply:
I'd have to dig a bit more into the code but can we ditch the batch ? Or retry or request it to be recreated ? 
I can see a cpu blip flipping a byte and the enclave panic'ing or something similar 🤔

Reply:
Worth doing I think? Or add a big todo to remove this panic in the future ?